From: Jo-Philipp Wich <jo@mein.io>
Date: Thu, 26 Jan 2017 15:30:00 +0100
Subject: [PATCH] config: adjust default values

Adjust the default interval settings to be better suited for slower embedded
devices.

Signed-off-by: Jo-Philipp Wich <jo@mein.io>

Date: Wed, 15 Apr 2020 22:35
Update by Twinzo1<1282055288@qq.com> on 2020-4-15 22:35

--- a/cfg/vnstat.conf
+++ b/cfg/vnstat.conf
<<<<<<< HEAD
@@ -95,13 +95,13 @@ YearlyYears    -1
 TopDayEntries  20
=======
@@ -98,13 +98,13 @@ DaemonGroup ""
 TimeSyncWait 5
>>>>>>> 82fdfd65b03d64d7c7f4cb057233f001619ac1b1
 
 # how often (in seconds) interface data is updated
-UpdateInterval 20
+UpdateInterval 60
 
 # how often (in seconds) interface status changes are checked
-PollInterval 5
+PollInterval 30
 
 # how often (in minutes) data is saved to database
-SaveInterval 5
+SaveInterval 30
 
 # how often (in minutes) data is saved when all interface are offline
 OfflineSaveInterval 30
